src/components/TopBar/CallButton.vue

```diff
@@ -132,6 +132,18 @@ export default {
 			return t('spreed', 'Start call')
 		},
 
+		startCallToolTip() {
+			if (this.callButtonTooltipText) {
+				return this.callButtonTooltipText
+			}
+
+			if (!this.conversation.canStartCall && !this.conversation.hasCall) {
+				return t('spreed', 'You can only join the call after a moderator started it.')
```

Comment:
Maybe it should be just _You can not start a call in this conversation_ or something like that, without any mention to moderators, as starting the calls could be limited only for guests but allowed for moderators and regular users too?

Reply:
Well for the guest this doesn't really matter

Reply:
And I guess they can figure they can't because the button is disabled.